cli/tests/test_cli.py

```
@@ -0,0 +1,1491 @@
+        mock_get_folders.return_value = (1, [{"id": 1, "name": "Global"}])
+        
+        # Mock CSV data with different type cases
+        mock_df = pd.DataFrame({
+            'name': ['Asset1', 'Asset2', 'Asset3'],
+            'type': ['primary', 'SUPPORT', 'Primary']  # Test case insensitivity
+        })
+        mock_read_csv.return_value = mock_df
+        
+        # Mock successful API responses
+        mock_post.side_effect = [
+            MagicMock(status_code=201, json=lambda: {"id": i}) for i in range(1, 4)
```

Ruff (0.11.9)

Function definition does not bind loop variable `i`

(B023)